components/training/eventBooking.tsx

```diff
@@ -177,6 +193,7 @@ export const eventBookingBlock = {
   eventBooking: "EventBooking",
   duration: "duration",
   price: "price",
+  earlyBirdPrice: "earlyBirdPrice",
```

Comment:
Shouldn't it be discounted price? "Early bird" text should be dynamically added

Reply:
For better customization, I changed earlyBirdPrice to discountPrice and changed its type into string as well